localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ rationalism:0 "Rationalism"
+ rationalism_adj:0 "Rational"
+ rationalism_adherent:0 "Rationalist"
+ rationalism_adherent_plural:0 "Rationalists"
+ rationalism_desc:0 "$rationalism_adj$ philosophy is focused on forward-thinking, inventive concepts, and little thought has ever been given to history or non-scientific records. $rationalism_adherent_plural$ value diligence and intelligence as they drive technological progress forward."
+ 
+ cult_of_wealth:0 "Fortunism"
+ cult_of_wealth_adj:0 "Fortunate"
+ cult_of_wealth_adherent:0 "Fortunist"
+ cult_of_wealth_adherent_plural:0 "Fortunists"
+ cult_of_wealth_desc:0 "$cult_of_wealth$ is an ideology based around the goal of making a profit. $cult_of_wealth_adherent_plural$ find themselves to be pragmatic, placing their faith in gold and their own abilities to hoard it. They dismiss insubstantial concepts such as shamanism and religion, preferring power and influence in measurable quantities that they can touch, see, and spend as their aspirations demand."
+ 
+ grond_group:0 "Grondism"
+ grond_group_adj:0 "Grondic"
+ grond_group_adherent:0 "Grondist"
+ grond_group_adherent_plural:0 "Grondists"
+ grond_group_desc:0 "A group of religions founded by descendants of Grond, a massive elemental giant created by the titan Aggramar. These religions exist at the intersection of Titan worship and shamanism."
+ 
+ gorgog_worship:0 "Gogism"
+ gorgog_worship_adj:0 "Gogic"
+ gorgog_worship_adherent:0 "Gogist"
+ gorgog_worship_adherent_plural:0 "Gogists"
+ gorgog_worship_desc:0 "Gorgog the Gronnslayer was the founder of the Gorian Empire. He broke the bodies of the gronn, opened their bones and ate their marrow to prove they were not gods. Arcane magic plays a big role in the $gorgog_worship_adj$ society. It is a great honor to be a mage."
+ 
+ # aviana_worship:0 "Avianism"
+ # aviana_worship_adj:0 "Avianic"
+ # aviana_worship_adherent:0 "Avianist"
+ # aviana_worship_adherent_plural:0 "Avianists"
+ # aviana_worship_desc:0 "A pantheon of gods led by Aviana, the mother of harpies, and mistress of the Mother Tree, G'Hanir."
+ 
+ shamanism_group:0 "Shamanism"
+ shamanism_group_adj:0 "Shamanic"
+ shamanism_group_adherent:0 "Shamanist"
+ shamanism_group_adherent_plural:0 "Shamanists"
+ shamanism_group_desc:0 "A wide group of beliefs built around the deification of natural phenomena, spirits and elementals."
+ 
+ orcish_shamanism:0 "Ancestrism"
+ orcish_shamanism_adj:0 "Ancestral"
+ orcish_shamanism_adherent:0 "Ancestral"
+ orcish_shamanism_adherent_plural:0 "Ancestrals"
+ orcish_shamanism_desc:0 "One of the main tenets of shamanism practices in the world. Most races respect the dead, but those who worship their ancestors, believe that the fallen possess powers, which the living can use. They speak to them and can request their help through dreaming and such. They carve memorials to the fallen dead and place stones around permanent burial grounds. There are many rituals one can perform, alone or in a group."
```

Made a few changes. Also, I would cut off the last two sentences. They are kinda out of place.
```suggestion
 orcish_shamanism_desc:0 "One of the main tenets of shamanism practices in the world. Most races respect the dead, but those who worship their ancestors believe that the fallen possess powers which the living can use. They speak to them and can request their help through dreaming and visions. They carve memorials to the fallen dead and place stones around permanent burial grounds. There are many rituals one can perform, alone or in a group."
```